BUILDING.md

```diff
@@ -280,6 +280,17 @@ When using the debug binary, core dumps will be generated in case of crashes.
 These core dumps are useful for debugging when provided with the 
 corresponding original debug binary and system information.
 
+Reading the core dump requires a gdb that was build on the same platform as the
+core dump was captured, so 64bit gdb if node was build as 64bit and Linux gdb if node
```

Comment:
`so 64bit gdb if node was build as 64bit and Linux gdb if node...` -> `(i.e. 64bit gdb for node built on a 64bit system, Linux gdb for node built on Linux)` wdyt?